mlrun/db/httpdb.py

```diff
@@ -574,7 +574,12 @@ def watch_log(self, uid, project="", watch=True, offset=0):
             else:
                 nil_resp += 1
 
-            if watch and state in ["pending", "running"]:
+            if watch and state in [
+                mlrun.runtimes.constants.RunStates.pending,
+                mlrun.runtimes.constants.RunStates.running,
+                mlrun.runtimes.constants.RunStates.created,
```

Comment:
im not sure created is a legit state to watch on as it might not change overtime but would remain as is
I would omit that

Reply:
But if the run state is `created` at the time of checking, the watch would stop before it even started running, which means that this won't wait for the run to be completed at all.